driver/longhorn

```diff
@@ -230,8 +230,10 @@ formatDevIfNeed() {
             echo -ne "Success"
         fi
     else
-        if ! OUT=$(mkfs -t ${newFsType} ${fullDevName} >/dev/null 2>&1); then
-            echo -ne "Failed to create fs ${newFsType} on device ${fullDevName}"
+        OUT=$(mkfs -t ${newFsType} ${fullDevName} >&1 2>&1)
+        if [ $?!=0 ]; then
+            OUT=$(echo "${OUT}" | tr "\n" " ")
```

Comment:
It's a nice touch you got here.

What's the result with `\n` looks like? Wondering how kubelet will show it. Also check the `kubectl describe <pod>` to see how it looks.